src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -150,7 +150,7 @@
 	<PackageReference Include="SharpDX.Mathematics" Version="4.2.0" />
 	<PackageReference Include="System.Reflection.MetadataLoadContext" Version="6.0.0" />
     <Reference Include="Dynamo.Microsoft.Xaml.Behaviors">
-      <HintPath>..\..\extern\Microsoft.Xaml.Behaviors\$(TargetFramework)\Dynamo.Microsoft.Xaml.Behaviors.dll</HintPath>
+      <HintPath>..\..\extern\Microsoft.Xaml.Behaviors\net6.0-windows\Dynamo.Microsoft.Xaml.Behaviors.dll</HintPath>
```

Comment:
??

Reply:
we do not have this dll built for net8. (i.e extern\Microsoft.Xaml.Behaviors\net8.0\Dynamo.Microsoft.Xaml.Behaviors.dll does not exist)
I am not convinced that we need to build specifically for net8, since it seems to work just fine.